### PR TITLE
Remove samples, remove plugin reprompt

### PR DESCRIPTION
**Pull request by rmberrios**
Remove samples, remove plugin reprompt

### CODE REVIEW

File: docs/account-linking.rst [hunk 1]
```diff
@@ -184,14 +184,12 @@ The serverless framework is a tool that helps you manage your lambda application
 Running the project
 ---------------------
 
-1. Clone the `Voxa <https://github.com/mediarain/voxa>`_ repository
+1. Clone the `Account Linking Sample <https://github.com/mediarain/voxa-accountlinking-sample>`_ repository
 
-2. Create a new skill project using the ``samples/starterKit`` directory as a basis
+2. Make sure you're running node 4.3, this is easiest with `nvm <https://github.com/creationix/nvm>`_
```
Comment (armonge) [inline docs/account-linking.rst]: can we update this to 6.10?

File: docs/my-first-podcast.rst [hunk 2]
```diff
@@ -205,24 +205,22 @@ serverless.yml
...
 Running the project
 ---------------------
 
-1. Clone the `Voxa <https://github.com/mediarain/voxa>`_ repository 
-
-2. Create a new skill project using the ``samples/my-first-podcast`` directory as a basis
+1. Clone the `Audio Podcast Sample <https://github.com/mediarain/voxa-audiopodcast-sample>`_ repository
 
-3. Make sure you're running node 4.3, this is easiest with `nvm <https://github.com/creationix/nvm>`_
+2. Make sure you're running node 4.3, this is easiest with `nvm <https://github.com/creationix/nvm>`_
```
Comment (armonge) [inline docs/my-first-podcast.rst]: same, let's update this

File: docs/starter-kit.rst [hunk 3]
```diff
@@ -87,13 +87,13 @@ serverless.yml
...
+
   $ sls deploy
-  
+
 Dependencies (install these first)
 ----------------------------------
 1. If you are using node.js for the first time, install node:
-  * Install `nvm <http://nvm.sh/>`_, ``curl -o- https://raw.githubusercontent.com/creationix/nvm/v0.33.1/install.sh | bash``, 
+  * Install `nvm <http://nvm.sh/>`_, ``curl -o- https://raw.githubusercontent.com/creationix/nvm/v0.33.1/install.sh | bash``,
   * Install node 4.3, ``nvm install 4.3``
```
Comment (armonge) [inline docs/starter-kit.rst]: same

**Comment by armonge** [comment]
Weird that coverage dropped when we're actually not removing any code or test that we track coverage of...

**Comment by rmberrios** [comment]
I had to drop a file, and the reference to the reprompt plugin. If you take that in consideration there are less line of codes to check. So the percentage would decreased

**Pull request merged**